ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -110,9 +110,9 @@ function getStatus(node: IronfishNode): GetStatusResponse {
   const status: GetStatusResponse = {
     peerNetwork: {
       peers: peers.length,
-      isReady: false,
-      inboundTraffic: 0,
-      outboundTraffic: 0,
+      isReady: node.peerNetwork.isReady,
+      inboundTraffic: Math.min(node.metrics.p2p_InboundTraffic.rate1s, 0),
+      outboundTraffic: Math.min(node.metrics.p2p_OutboundTraffic.rate1s, 0),
```

Comment:
Should this be `Math.max` if the rates are positive?

Reply:
Opps, thanks.